driver-core/src/main/com/mongodb/client/model/search/SearchConstructibleBsonElement.java

```diff
@@ -88,6 +88,11 @@ public TextSearchOperator synonyms(final String name) {
         });
     }
 
+    @Override
+    public RegexSearchOperator allowAnalyzedField(final boolean isAllowed) {
+        return newWithAppendedValue("allowAnalyzedField", notNull("isAllowed", isAllowed));
+    }
```

Comment:
```suggestion
    public RegexSearchOperator allowAnalyzedField(final boolean allowAnalyzedField) {
        return newWithAppendedValue("allowAnalyzedField", allowAnalyzedField);
    }
```
Redundant null check. For naming, a similar method uses the same param name as the method itself:

```
    public SearchOptions returnStoredSource(final boolean returnStoredSource) {
        return newAppended("returnStoredSource", new BsonBoolean(returnStoredSource));
    }
```

This can be changed in the interfaces as well.

Reply:
Got it.